packages/presentation/src/components/DrawingBoard.svelte

```diff
@@ -0,0 +1,147 @@
+<!--
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { Button, IconDelete, IconEdit, resizeObserver } from '@hcengineering/ui'
+  import { drawing, type DrawingData, type DrawingTool } from '../drawing'
+  import IconEraser from './icons/Eraser.svelte'
+
+  export let active = false
+  export let readonly = false
+  export let imageWidth: number | undefined
+  export let imageHeight: number | undefined
+  export let drawingData: DrawingData
+  export let saveDrawing: (data: any) => Promise<void>
+
+  let drawingTool: DrawingTool = 'pen'
+  let penColor = 'blue'
+  const penColors = ['red', 'green', 'blue', 'white', 'black']
```

Comment:
We have colors.ts with a lot of color constants, better to use colors from it.

Reply:
we don't have colors like wite or black in our palette, only some weird colors :)

Reply:
This is just random start color set, they fit for any theme. I suppose, eventually there should be a color selector dialog and a dynamic color set of the most recently used colors.